README.md

```diff
@@ -43,13 +53,16 @@ export HISTSIZE=0
 #Delete cache of yum
 yum clean all
 rm -rf /var/cache/yum
-#Optimize the area of ​​the virtual hard disk
+#Optimize the area of the virtual hard disk
 dd if=/dev/zero of =/ZERO bs=1M
 rm -f /ZERO
 shutdown -h now
 ```
+Update January 2019: (Sam Anthony) I could not seem to get the `dd` command to work.
```

Comment:
I just tested it and indeed the space after `of` breaks it but when I removed that it seems to work fine.

```
[ec2-user@ip-10-204-90-168 ~]$ sudo dd if=/dev/zero of =/ZERO bs=1M
dd: unrecognized operand ‘of’
Try 'dd --help' for more information.
[ec2-user@ip-10-204-90-168 ~]$ sudo dd if=/dev/zero of=/ZERO bs=1M
dd: error writing ‘/ZERO’: No space left on device
7015+0 records in
7014+0 records out
7354712064 bytes (7.4 GB) copied, 113.948 s, 64.5 MB/s
[ec2-user@ip-10-204-90-168 ~]$ cat /etc/*release*
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
HOME_URL="https://amazonlinux.com/"
Amazon Linux release 2 (Karoo)
cpe:2.3:o:amazon:amazon_linux:2
[ec2-user@ip-10-204-90-168 ~]$
```